Config/BaseProgressionSystem.ini

```diff
@@ -1,6 +1,6 @@
 ﻿[/Script/ProgressionSystemRuntime.PSWorldSubsystem]
-PSDataAssetInternal=/ProgressionSystem/DataAssets/DA_ProgressionSystem.DA_ProgressionSystem
+DataAssetInternal=/ProgressionSystem/DataAssets/DA_ProgressionSystem.DA_ProgressionSystem
 
 ; Increment save file version extennsion to force users run empty save
 ; Increment whenever a major changes done with safe file structure
-SaveFileVersionExtensionInternal=3
\ No newline at end of file
+SaveFileVersionExtensionInternal=4
```

Comment:
Before continue reviewing, please sync to latest PS-develop and resolve conflicts